talking-about-their-pronouns.md

```diff
@@ -0,0 +1,38 @@
+# Asking someone's pronouns
+
+## Over Slack / in conversation
+
+What: Sorry, just to check so I can get them right, do you mind me asking what your pronouns are?
+
+# Correcting someone who misgenders someone else
+
+## During a meeting
+
+### Respond in the moment
+
+When: Once the person who's just got the pronouns incorrect has finished talking
+How: Verbally
+What: Just to point out, I'm pretty sure their pronouns are they/them
+
+### Respond async
+
+Where: Slack DM, email etc.
+What: FYI, I think you used "she" to refer to Sarah and I'm pretty sure their pronouns are they/them :)
```

Comment:
```suggestion
When: As soon as it happens
What: FYI, Kieran's pronouns are they/them - I saw it in their Slack profile / they told me last week/month/when we met :)
```

a) I'd prefer we didn't embed the wrong pronouns by mentioning them! :sweat_smile: 
b) It gives them a tool for getting things right for that person, and others, in future.